README.md

```diff
@@ -57,10 +60,86 @@ authentication token that you can get [here](https://console.redhat.com/openshif
 If this attribute isn't used then the provider will try to get the token it from
 the `OCM_TOKEN` environment variable.
 
+### Create AWS Rosa STS Cluster
+
+The following example shows a production grade rosa cluster with:
+
+* Existing VPC & Subnets
+* Multi AZ
+* Proxy
+* STS
+
+```
+locals {
+  sts_roles = {
```

Comment:
`sts_roles` looks good for a first pass, but there will be issues down the line (I'm thinking upgrades) if we don't enforce consistent naming. I think we should eventually only expose a prefix for the account roles (defaults to `ManagedOpenShift`) and a prefix for the operator roles (defaults to '${cluster.name}-${random-hash}`). The rest of the data in this entire structure can be deduced from those two components along with a few API calls to OCM.